packages/react-components/src/organisms/ManuscriptVersionCard.tsx

```diff
@@ -403,7 +408,7 @@ const ManuscriptVersionCard: React.FC<ManuscriptVersionCardProps> = ({
                       <Suspense fallback={<Loading />}>
                         <Discussion
                           modalTitle="Reply to quick check"
-                          canReply={isActiveManuscript && isTeamMember}
+                          canReply={isActiveVersion && canUpdateDiscussion}
```

Comment:
`canUpdateDiscussion` already takes `isActiveVersion` into consideration :)
```suggestion
                          canReply={canUpdateDiscussion}
```

Reply:
Good catch, thanks! 👌🏾